projects/plugins/jetpack/_inc/client/newsletter/index.jsx

```diff
@@ -48,6 +49,7 @@ function Subscriptions( props ) {
 				<SubscriptionsSettings siteRawUrl={ siteRawUrl } blogID={ blogID } />
 			) }
 			{ isNewsletterCategoriesEnabled && <NewsletterCategories /> }
```

Comment:
I think as per the design we will want to have the message above the Newsletter Categories, right?
(p1HpG7-rkO-p2)

Reply:
True. I'll change it.

Reply:
True. Updated.